spr/src/message.rs

```diff
@@ -326,22 +343,21 @@ mod tests {
 // Reviewer:    a, b, c"#,
 r#"Hello
 
-Test-Plan: testzzz
-
-Summary:
-here is
+Here is
```

Comment:
(feel free to ignore)

changing caps here isn't really relevant to the change, is it?